### PR TITLE
Documentation update for splitting gpslogger into a component with a device tracker platform

### PR DESCRIPTION
**Pull request by rohankapoorcom**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20044

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/device_tracker.gpslogger.markdown [hunk 1]
```diff
@@ -11,63 +11,10 @@ ha_category: Presence Detection
...
-If your battery drains too fast then you can tune the performance of GPSLogger under **Performance** -> **Location providers**
-
-<p class='img'>
-  <img width='300' src='/images/components/gpslogger/performance.png' />
-  Performance
+<p class='note'>
+You must have the [Geofency component](/components/geofency/) configured to use this device tracker.
 </p>
 
 A request can be forced from the app to test if everything is working fine. A successful request will update the `known_devices.yaml` file with the device's serial number.
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.gpslogger.markdown]: Leaving this here looks out of place. It's already copied to the component.

Comment (rohankapoorcom) [reply source/_components/device_tracker.gpslogger.markdown]: Removed.

**Review by MartinHjelmare — APPROVED** [review]

File: source/_components/device_tracker.gpslogger.markdown [hunk 2]
```diff
@@ -11,63 +11,8 @@ ha_category: Presence Detection
 ha_release: 0.34
 ---
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.gpslogger.markdown:12]: We should change the title of this page to distinguish it from component. Right now that's impossible in the component search list in the docs.

File: source/_components/device_tracker.gpslogger.markdown [hunk 3]
```diff
@@ -11,63 +11,8 @@ ha_category: Presence Detection
...
-- Add `Authorization: Bearer LLAT` to the HTTP Headers setting (replace `LLAT` with your Long Lived Access Token).
-- You can change the name of your device name by replacing `&device=%SER` with `&device=DEVICE_NAME`.
-
-If your battery drains too fast then you can tune the performance of GPSLogger under **Performance** -> **Location providers**
-
-<p class='img'>
-  <img width='300' src='/images/components/gpslogger/performance.png' />
-  Performance
+<p class='note'>
+You must have the [Geofency component](/components/geofency/) configured to use this device tracker.
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.gpslogger.markdown]: Stale copy paste from the geofency component.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Comment by MartinHjelmare** [comment]
You can preview the PR in the netlify deploy link in the build checks.

File: source/_components/device_tracker.gpslogger.markdown (hunk 2 above)
Comment (rohankapoorcom) [reply source/_components/device_tracker.gpslogger.markdown:12]: Fixed.

File: source/_components/device_tracker.gpslogger.markdown (hunk 3 above)
Comment (rohankapoorcom) [reply source/_components/device_tracker.gpslogger.markdown]: This is what I get for doing too many of these in quick succession... should be all cleaned up now.

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
There's no logo for these pages. Should we add one?

**Comment by rohankapoorcom** [comment]
Let's do it in the next pull request (coming along with https://github.com/home-assistant/home-assistant/pull/20079) if that's okay?

**Pull request merged**